modules/config-fields-oauth.adoc

```diff
@@ -24,37 +25,66 @@
  + 
 **Example:** `https://api.github.com/`
 | {nbsp}{nbsp}{nbsp}**.CLIENT_ID** + 
-{nbsp}{nbsp}{nbsp}(Required) | String |   The registered client ID for this {productname} instance; cannot be shared with GITHUB_TRIGGER_CONFIG + 
+{nbsp}{nbsp}{nbsp}(Required) | String |   The registered client ID for this {productname} instance; cannot be shared with `GITHUB_TRIGGER_CONFIG`. + 
  + 
 **Example:** `0e8dbe15c4c7630b6780`
 | {nbsp}{nbsp}{nbsp}**.CLIENT_SECRET** + 
-{nbsp}{nbsp}{nbsp}(Required)  | String | The registered client secret for this {productname} instance + 
+{nbsp}{nbsp}{nbsp}(Required)  | String | The registered client secret for this {productname} instance. + 
  + 
 **Example:** `e4a58ddd3d7408b7aec109e85564a0d153d3e846`
 | {nbsp}{nbsp}{nbsp}**.GITHUB_ENDPOINT**  + 
-{nbsp}{nbsp}{nbsp}(Required) | String |  The endpoint for GitHub (Enterprise) + 
+{nbsp}{nbsp}{nbsp}(Required) | String |  The endpoint for GitHub (Enterprise). + 
  + 
 **Example**: `https://github.com/`
 | {nbsp}{nbsp}{nbsp}**.ORG_RESTRICT** | Boolean | If true, only users within the organization whitelist can login using this provider.
 |===
 
-
+[id="google-oauth-config-fields"]
 == Google OAuth configuration fields
 
 .Google OAuth fields
 [cols="3a,1a,2a",options="header"]
 |===
 | Field | Type | Description
-| **FEATURE_GOOGLE_LOGIN** | Boolean | Whether Google login is supported + 
+| **FEATURE_GOOGLE_LOGIN** | Boolean | Whether Google login is supported. + 
  + 
 **Default: `False`
-| **GOOGLE_LOGIN_CONFIG** | Object | Configuration for using Google for external authentication
+| **GOOGLE_LOGIN_CONFIG** | Object | Configuration for using Google for external authentication.
 | {nbsp}{nbsp}{nbsp}**.CLIENT_ID** + 
-{nbsp}{nbsp}{nbsp}(Required)  | String | The registered client ID for this {productname} instance + 
+{nbsp}{nbsp}{nbsp}(Required)  | String | The registered client ID for this {productname} instance. + 
  + 
 **Example:** `0e8dbe15c4c7630b6780`
 | {nbsp}{nbsp}{nbsp}**.CLIENT_SECRET** + 
-{nbsp}{nbsp}{nbsp}(Required)  | String | The registered client secret for this {productname} instance + 
+{nbsp}{nbsp}{nbsp}(Required)  | String | The registered client secret for this {productname} instance. + 
  + 
-**Example:** e4a58ddd3d7408b7aec109e85564a0d153d3e846
-|===
\ No newline at end of file
+**Example:** `e4a58ddd3d7408b7aec109e85564a0d153d3e846`
+|===
+
+[id="misc-oauth-config-fields"]
+== Miscallaneous OAuth configuration fields
+
+.Miscallaneous OAuth fields
```

Comment:
Typo s/Miscallaneous/Miscellaneous/

Anyhow, @stevsmit shouldnt this be ".OIDC fields" instead of ".Misc OAuth fields"?
I think this is about authentication (OIDC) and not authorization (OAuth), but I might be wrong...